env.dist

```diff
@@ -112,9 +112,12 @@ export PROXY_KEY="0x24FEEDFACEDEADBEEFCAFE"
 export PROXY_HOSTNAME="http://localhost:8081"
 
 ##
-# Config for Grunt tasks (see gruntfile.js)
+# Bramble config, including grunt tasks (see gruntfile.js)
 #
 
+## Where is bramble being loaded from?
+export BRAMBLE_URI="http://localhost:8888"
```

Comment:
Do the servers above use port 8888 or 8000?  I think it's the latter, but confirm.